src/hooks/use-suggestions.ts

```diff
@@ -272,12 +272,24 @@ export const useSuggestions = (
                     if (isObject(fieldOf) && !Array.isArray(fieldOf)) {
                         for (const key of Object.keys(fieldOf)) {
                             if (key === '*') continue
+                            if (aliasedKeys.includes(`${matchedField}.${key}`))
```

Comment:
How does it work when the attributes have mutual strings that they start with ? Would the suggestion be correct ? Maybe should regex it as a starter or have a point before the `matchedField`, and check if it;s the end after `key`.

For exmaple `aaa.bbb` and `aa.bb`

Reply:
this does not change how it works, it just skips the keys for which we have aliases. if there is no alias, the condition fails and the code proceeds as before